include/deal.II/base/parameter_handler.h

```diff
@@ -1558,7 +1558,35 @@ class ParameterHandler : public Subscriptor
     ShortText = 193
   };
 
+  /**
+   * Flags to control input file scanning behavior.
+   */
+  enum ReadInputFlags
+  {
+    /**
+     * A flag for no flag.
+     */
+    ReadInputFlag_Null = 0,
```

Comment:
I can imagine put the enum into a namespace but not a struct. Can you show me an example at somewhere else in deal.II?